golang/cmd/kafka-to-postgresql-v2/postgresql/analytics_test.go

```diff
@@ -183,9 +235,26 @@ func TestProductType(t *testing.T) {
 
 		// Expect Exec from InsertProductTypeCreate
 		mock.ExpectBeginTx(pgx.TxOptions{})
-		mock.ExpectExec(`INSERT INTO product_type\(external_product_type_id, cycle_time_ms, asset_id\)
-		VALUES \(\$1, \$2, \$3\)
-		ON CONFLICT \(external_product_type_id, asset_id\) DO NOTHING`).
+		mock.ExpectExec(`
+			INSERT INTO product_type
+            \(
+                        external_product_type_id,
+                        cycle_time_ms,
+                        asset_id
+            \)
+            VALUES
+            \(
+                        \$1,
+                        \$2,
+                        \$3
+            \)
+			on conflict
```

Comment:
this needs to be reconsidered in https://github.com/united-manufacturing-hub/MgmtIssues/issues/1671

(linking it here, so we know it needs to be done in https://github.com/united-manufacturing-hub/MgmtIssues/issues/1671)